tools/test-js-with-node

```diff
@@ -93,6 +93,12 @@ os.environ['TZ'] = 'UTC'
 
 INDEX_JS = 'frontend_tests/zjsunit/index.js'
 
+# Add ".js" to the end of all the file arguments, so index.js
+# can actually verify if the file exists or not.
+for args in options.args:
```

Comment:
say `arg` not `args`, and you you can say `if not arg.endswith('.js')`

Reply:
Ah, yes updated.